src/user/room.ts

```diff
@@ -428,23 +428,31 @@ export class Room extends Accessory implements Sayable {
     textOrContactOrFileOrUrl : string | Contact | FileBox | UrlLink,
     mention?                 : Contact | Contact[],
   ): Promise<void> {
+
+    const replyToList: Contact[] = Array.isArray(mention)
```

Comment:
Change this line to:

```
const replyToList: Contact[] = [...(mention || [])]
```

Reply:
I don't think that works because the ```mention``` may be  ```Contact```, and I believe we don't want to have ```list of list(2D array)```  here
![image](https://user-images.githubusercontent.com/25014776/46369665-7482d680-c638-11e8-84f9-a675ed7b227c.png)